aggregation/quantile/cm/options.go

```diff
@@ -61,9 +62,10 @@ type options struct {
 // NewOptions creates a new options
```

Comment:
Yeah I'm gonna put up a separate PR that does a blanket sweep across the entire m3aggregator repo looking for comments missing periods.